components/webui/imports/ui/SearchView/SearchResults/SearchResultsTable/index.jsx

```diff
@@ -143,20 +139,23 @@ const SearchResultsTable = ({
                                 <pre className={"search-results-content search-results-message"}>
                                     {result.message}
                                 </pre>
+                                {isExtractIrSupported &&
+                                    <div className={"search-results-file-link"}>
+                                        <FontAwesomeIcon icon={faFileLines}/>
+                                        {" "}
+                                        <a
+                                            className={"search-results-file-link"}
+                                            rel={"noopener noreferrer"}
```

Comment:
What's the reasoning behind setting these? Fwiw, `_blank` seems to [imply](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#target) `noopener` now:

> Note: Setting target="_blank" on <a> elements implicitly provides the same rel behavior as setting [rel="noopener"](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel/noopener) which does not set window.opener.

Reply:
Ah.. I didn't read the docs when I wrote this. 

Apparently the behaviour change is fairly recent (with Chrome adapting it only since 2021) so we might better keep the `noopener`:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#browser_compatibility:~:text=target%3D%22_blank%22%20implies%20rel%3D%22noopener%22%20behavior